src/components/SearchForm.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+
+function SearchForm(props) {
+    const [query, setQuery] = useState(props.searchQuery);
+
+    return (
+        <div>
+            <h1>FIND YOUR MOVIE</h1>
+            <input 
+                type="search" 
+                placeholder="What do you want to watch?" 
+                size="30" 
+                value={query}  
+                onChange={e => setQuery(e.target.value)}
+                onFocus={props.onSearch(query)}></input>
```

Comment:
It's necessary to trigger the callback when the user **presses Enter** or when the user **clicks the Search button**. 
Currently, it is triggered only when I focus on input. Please, review the handlers you used. This topic will help https://ru.reactjs.org/docs/forms.html